bit_manipulation/count_number_of_one_bits.py

```diff
@@ -1,19 +1,51 @@
-def get_set_bits_count(number: int) -> int:
+from timeit import timeit
+
+
+def get_set_bits_count_using_brian_kernighans_algorithm(number: int) -> int:
     """
     Count the number of set bits in a 32 bit integer
-    >>> get_set_bits_count(25)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(25)
     3
-    >>> get_set_bits_count(37)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(37)
     3
-    >>> get_set_bits_count(21)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(21)
     3
-    >>> get_set_bits_count(58)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(58)
     4
-    >>> get_set_bits_count(0)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(0)
     0
-    >>> get_set_bits_count(256)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(256)
     1
-    >>> get_set_bits_count(-1)
+    >>> get_set_bits_count_using_brian_kernighans_algorithm(-1)
+    Traceback (most recent call last):
+        ...
+    ValueError: the value of input must be positive
```

Comment:
Zero is not positive.
```suggestion
    ValueError: the value of input must not be negative
```